2-WebApp-graph-user/2-1-Call-MSGraph/Controllers/HomeController.cs

```diff
@@ -6,7 +6,7 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Options;
 using Graph=Microsoft.Graph;
-using Microsoft.Identity.Web.Client;
+using Microsoft.Identity.Web;
```

Comment:
In the library, aligned the folder structure and the namespaces (like @MarkZuber had proposed in the extension library)
Note that the library is not only for clients, but also resources / Web APIs (issuer validator for instance) and therefore the root namespace is not Microsoft.Identity.Client, but Microsoft.Identity.Web